### PR TITLE
docs: Fetching translation from Weblate reworked

### PR DESCRIPTION
**Pull request by comradekingu**
## Checklist

- [ ] Lint and unit tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added documentation to describe my feature.
- [ ] I have squashed my commits into logic units.
- [ ] I have described the changes in the commit messages.

### CODE REVIEW

File: docs/devel/integration.rst [hunk 1]
```diff
@@ -119,21 +121,21 @@ You can add new strings in Weblate with :ref:`component-manage_units` turned
 on, but it is usually better to introduce new strigs together with the code
 changes that introduced them.
 
-Monolingual formats need addition of the new string to
-:ref:`component-template`. This is typically done by the developers during
-developing the code. You might want to introduce review of those strings using
+Monolingual formats need to be configured so that new strings are added to
+:ref:`component-template`. This is typically done by developers as they
+develop the code. You might want to introduce review of those strings using
```
Comment (comradekingu) [inline docs/devel/integration.rst:126]:
```suggestion
develop the code.
You might want to introduce review of those strings in Weblate using
```
?

File: docs/devel/integration.rst [hunk 2]
```diff
@@ -119,21 +121,21 @@ You can add new strings in Weblate with :ref:`component-manage_units` turned
...
 needed to update existing translations, see :ref:`updating-target-files`.
 
 .. hint::
 
-   You might want to integrate this into your contrinuous integration pipelines
-   to make new strings automatically appear for translation. Such pipeline
-   should also cover :ref:`avoid-merge-conflicts`.
+   Integrate this into your contrinuous-integration pipelines
+   if you want new strings to appear automatically for translation.
+   Such a pipeline should also cover :ref:`avoid-merge-conflicts`.
```
Comment (comradekingu) [inline docs/devel/integration.rst:138]: Does this means the user is covered, or need to attend to it themselves?

File: docs/devel/integration.rst [hunk 3]
```diff
@@ -22,19 +23,19 @@ Fetching updated translations from Weblate
 ++++++++++++++++++++++++++++++++++++++++++
 
 Weblate stores updated strings in a database and commits them to a local
-version control repository. You can add Weblate repository (when
-:ref:`git-exporter` is turned on) as additional remote and fetch translations
-update from it.
+version-control repository. You can add a Weblate repository (when
+:ref:`git-exporter` is turned on) as an additional remote repository
+and fetch translations updates from it.
```
Comment (comradekingu) [inline docs/devel/integration.rst:28]:
```suggestion
and fetch translation updates from it.
```

File: docs/devel/integration.rst [hunk 4]
```diff
@@ -119,21 +121,21 @@ You can add new strings in Weblate with :ref:`component-manage_units` turned
 on, but it is usually better to introduce new strigs together with the code
```
Comment (comradekingu) [inline docs/devel/integration.rst:121]:
```suggestion
on, but it is usually better to introduce new strings together with the code
```

File: docs/devel/integration.rst [hunk 5]
```diff
@@ -116,24 +118,24 @@ Introducing new strings
 +++++++++++++++++++++++
 
 You can add new strings in Weblate with :ref:`component-manage_units` turned
```
Comment (comradekingu) [inline docs/devel/integration.rst:120]: Does having it on allow adding strings (correct now), or does that add strings btw.?

"When it is on"? Is it off by default? Manually? Not a very strong sentence.

File: docs/devel/integration.rst [hunk 6]
```diff
@@ -143,18 +145,17 @@ needed to update existing translations, see :ref:`updating-target-files`.
...
 .. _manage-vcs:
 
-Managing version control repository
-+++++++++++++++++++++++++++++++++++
+Managing the local VCS repository
++++++++++++++++++++++++++++++++++
 
-Weblate stores all translation the version control repository. It can be either
-connected to upstream one, or it can be only internal. The :guilabel:`Repository
-maintenance` lets you manipulate with the repository.
```
[the remark this answers is not in the document]
Comment (orangesunny) [reply docs/devel/integration.rst:156]: _managing_ or _managing manually_ is better. Thanks!

**Pull request merged**